librsync.yaml

```diff
@@ -0,0 +1,118 @@
+package:
+  name: librsync
+  version: 2.3.4
+  epoch: 0
+  description: Library for delta compression of streams
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/librsync/librsync
+      tag: v${{package.version}}
+      expected-commit: e364852674780e43d578e4239128ff7014190ed3
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_BUILD_TYPE=Release
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: librsync-dev
+    pipeline:
+      - uses: split/dev
+    test:
+      pipeline:
```

Comment:
for `-dev` packages we can use `test/pkgconf` if there's a `*.pc` file in here and `test/tw/header-check` and `test/tw/ldd-check`